src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -59,7 +52,7 @@ func NewClient(addr, appName string) *MobileClient {
 
 	hqfd := client.Configure(clientConfig.Client)
 
-	hqfdc := hqfd.DirectHttpClient()
+	hqfdc := hqfd.NewDirectDomainFronter()
```

Comment:
I agree.  I didn't change it (yet) because I hadn't made sure that was the case and that is orthogonal to using DDF for fetching the config.  Do you want me to change this for this PR, or shall we just open an issue to do a different one for that?